pom.xml

```diff
@@ -10,7 +10,7 @@
   </parent>
 
   <artifactId>plexus-archiver</artifactId>
-  <version>3.5.1-SNAPSHOT</version>
+  <version>3.6-SNAPSHOT</version>
```

Comment:
Can we do 3.6.0-SNAPSHOT instead ? I would suggest to have three digits.. ? WDYT ?

Reply:
The last couple of releases are two digits. I don't know if there is any particular reason - I think @michael-o did the last release. Maybe he knows. Other than that I agree that with three digits is better.

Reply:
We have agreed  at some point to use semver with three digits.

Reply:
ok, then I'll change it to 3.6.0